src/ui/common/Header/HeaderComponent.tsx

```diff
@@ -0,0 +1,53 @@
+import AppBar from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import IconButton from '@mui/material/IconButton';
+import MenuIcon from '@mui/icons-material/Menu';
+import Badge from '@mui/material/Badge';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+import { drawerWidth } from '../../../styles/sidebarStyle';
+
+interface Props {
+  isOpened : boolean,
+  handleSidebarClose : () => void,  
+}
+
+
+const HeaderComponent = (props: Props) => 
+{
+    return (
+        <AppBar position= "static"
+         sx = {{...(props.isOpened && {
+            width: `calc(100% - ${drawerWidth}px)`,
+            marginLeft: `${drawerWidth}px`,
+          }),
+        }
+        }
```

Comment:
formatting should be }}

Reply:
Formatting has been improved using Prettier